schema.graphql

```diff
@@ -615,11 +616,10 @@ type PendingItem @key(fields: "url") {
   status: PendingItemStatus
 }
 
-type Item @key(fields: "givenUrl") {
+type Item @key(fields: "givenUrl") @key(fields: "itemId") {
   "key field to identify the Item entity in the Parser service"
   givenUrl: Url!
-  itemId: String! @shareable
-
+  itemId: String!
```

Comment:
curious - does this service need to specify/resolve the `itemId` field? if so, why? i'm having trouble finding docs on if a subgraph extending an entity has to define all keys for that entity.

to verify - is this subgraph is no longer responsible/capable of resolving either key on `Item`? or, can only one key be provided in the `Item` representation, and then this subgraph has to resolve the other?

Reply:
@jpetto form my understanding it should be able to resolve by both keys, whichever is provided